content/_data/ccs.yaml

```diff
@@ -7,6 +7,7 @@ fades: 0.75
 colors:
   - prime
   - accent
+  - special
```

Comment:
What does this do?

Reply:
We added a third color to CCS a while back, and this makes nunjucks aware of it – currently just used for the css previews we show.

Reply:
Which css previews?

Reply:
sorry: cascading-color previews, on the cascading-colors page